src/cpp/client.cpp

```diff
@@ -665,6 +665,62 @@ void Client::use_tensor_ensemble_prefix(bool use_prefix)
     this->_use_tensor_prefix = use_prefix;
 }
 
+parsed_map Client::get_db_node_info(std::string address)
+{
+    std::string host = address.substr(0, address.find(":"));
+    uint64_t port = std::stoul (address.substr(address.find(":") + 1),
+                                nullptr, 0);
+    Command cmd;
+    cmd.set_exec_address_port(host, port);
+    cmd.add_field("INFO");
+    CommandReply reply = this->_run(cmd);
+    // Put in a check here to make sure it is a str field or
+	// or throw error? this is done in reply.str()
```

Comment:
We can take this comment out.  ``CommandReply::str()`` will throw an error if the reply is not a string type.